seatunnel-connectors-v2/connector-flink-table-store/src/main/java/org/apache/seatunnel/connectors/seatunnel/fts/utils/RowConverter.java

```diff
@@ -0,0 +1,190 @@
+                    objects[i] = rowData.getFloat(i);
+                    break;
+                case DOUBLE:
+                    objects[i] = rowData.getDouble(i);
+                    break;
+                case DECIMAL:
+                    SeaTunnelDataType<?> decimalType = seaTunnelRowType.getFieldType(i);
+                    objects[i] =
+                            rowData.getDecimal(
+                                    i,
+                                    ((DecimalType) decimalType).getPrecision(),
+                                    ((DecimalType) decimalType).getScale());
+                    break;
+                case STRING:
+                    objects[i] = rowData.getString(i).toString();
+                    break;
+                case BOOLEAN:
+                    objects[i] = rowData.getBoolean(i);
+                    break;
+                case BYTES:
+                    objects[i] = rowData.getBinary(i);
+                    break;
+                case DATE:
+                    objects[i] = LocalDate.ofEpochDay(rowData.getInt(i));
+                    break;
+                case TIMESTAMP:
+                    // Now SeaTunnel not supported assigned the timezone for timestamp,
+                    // so we use the default precision 6
+                    TimestampData timestamp = rowData.getTimestamp(i, 6);
+                    objects[i] = timestamp.toLocalDateTime();
+                    break;
+                case ARRAY:
+                    SeaTunnelDataType<?> arrayType = seaTunnelRowType.getFieldType(i);
+                    ArrayData array = rowData.getArray(i);
+                    objects[i] = convert(array, arrayType);
+                    break;
+                case MAP:
+                    SeaTunnelDataType<?> mapType = seaTunnelRowType.getFieldType(i);
+                    MapData map = rowData.getMap(i);
+                    ArrayData keyArray = map.keyArray();
+                    ArrayData valueArray = map.valueArray();
+                    SeaTunnelDataType<?> keyType = ((MapType<?, ?>) mapType).getKeyType();
+                    SeaTunnelDataType<?> valueType = ((MapType<?, ?>) mapType).getValueType();
+                    Object[] key = (Object[]) convert(keyArray, keyType);
+                    Object[] value = (Object[]) convert(valueArray, valueType);
+                    Map<Object, Object> mapData = new HashMap<>();
+                    for (int j = 0; j < key.length; j++) {
+                        mapData.put(key[j], value[j]);
+                    }
+                    objects[i] = mapData;
+                    break;
+                case ROW:
+                    SeaTunnelDataType<?> rowType = seaTunnelRowType.getFieldType(i);
+                    RowData row = rowData.getRow(i, ((SeaTunnelRowType) rowType).getTotalFields());
+                    objects[i] = convert(row, (SeaTunnelRowType) rowType);
+                    break;
+                default:
```

Comment:
Adds the `MultiSet` type?

Reply:
In seatunnel inner datatype I can not find a suitable data type to adopt `MultiSet`, seatunnel support the following types:

![image](https://user-images.githubusercontent.com/51053924/226408817-a11cde05-8836-4b17-ab02-be4eb8737018.png)

Do you have any good advice?

Reply:
You can use this mapping:
Multiset -> Map<Element, Integer>